song_translator/track/urls.py

```diff
@@ -3,19 +3,20 @@
 from django.views.decorators.csrf import csrf_exempt
 
 from .views import TrackList, TrackDetail, SingerList, SingerDetail, translate_detail, TranslationList, \
-    signup, sign_in
+    signup, sign_in, CommentList, CommentDetail
 
 
 urlpatterns = [
     path('tracks/', TrackList.as_view()),
-    path('tracks/<int:pk>/', TrackDetail.as_view()),
-    path('tracks/<int:pk>/translations/', TranslationList.as_view()),
-    path('tracks/<int:pk>/translations/<int:transl_id>/', translate_detail),
-    path('singers/', csrf_exempt(SingerList.as_view())),
-    path('singers/<int:pk>/', SingerDetail.as_view()),
-    path('users/', signup),
-    path('users/login/', sign_in),
-
+    path('tracks/<int:pk>', TrackDetail.as_view()),
```

Comment:
why you change slash? I believe there's django middleware that's taking care of it, so it shouldn't matter

Reply:
ok, I just read a lot of opinions about this question and got a little confused)